src/components/Form/HorizontalFieldText/index.tsx

```diff
@@ -98,7 +98,7 @@ export class HorizontalFieldText extends React.Component<HorizontalFieldTextProp
                   data-for="horizontal-field-text-label__tooltip"
                   data-tip={tooltip}
                 />
-                <ReactTooltip effect="solid" id="horizontal-field-text-label__tooltip" />
+                <ReactTooltip effect="solid" id="horizontal-field-text-label__tooltip" html={true} />
```

Comment:
based on [react tooltip docs](https://github.com/wwayne/react-tooltip#jsx-note), i think you should wrap the tooltip with renderToString from react-dom/server. you can refer to AP to see renderToString implementation

Reply:
other components should have it too

Reply:
my implementation [here](https://github.com/AccelByte/justice-ui-library/pull/7/files#diff-5d13bb9b12c8111037f24b53006ae789d728b3c02791b9ab51571c3d93b410a8R80)